app/controllers/application_controller.rb

```diff
@@ -21,7 +21,11 @@ def redirect_after_login
     elsif session[:return_to]
       redirect_to session[:return_to]
     else
-      OrdersController.create
+      @order = Order.create(user_id: current_user.id, status: "ordered")
+      @order.create_order_items(@cart)
+      @order.update_attributes(total_price: @order.order_total)
+      @cart.clear
+      redirect_to order_path(@order)
```

Comment:
yes, this allows a user not to be redirected back to the checkout page if that is how they are prompted to login. This works now, but will add to the waffle list to add a checkout summary page which will be a much cleaner way to handle this.